e2e/rm_test.go

```diff
@@ -430,10 +430,10 @@ func TestRemoveLocalDirectory(t *testing.T) {
 
 	assertLines(t, result.Stdout(), map[int]compareFunc{
 		0: equals(""),
-		1: equals("rm file1.txt"),
-		2: equals("rm file2.txt"),
-		3: equals("rm readme.md"),
-	}, sortInput(true))
+		1: equals("rm testdir/file1.txt"),
+		2: equals("rm testdir/file2.txt"),
+		3: equals("rm testdir/readme.md"),
+	}, strictLineCheck(true), sortInput(true))
```

Comment:
we can remove strictLineCheck(true). it's enabled by default.

Reply:
done